docs/modules/user-guide/nav.adoc

```diff
@@ -31,6 +31,10 @@
 
 * Registry Administrator
 ** xref:devfile-registry.adoc[]
```

Comment:
I'm working on getting the `devfile registry` to render properly. Right now it does not. 
<img width="363" alt="Screen Shot 2021-09-30 at 12 47 21 PM" src="https://user-images.githubusercontent.com/70717303/135497244-4579431f-a514-4bd3-b84e-3824bff35b97.png">

Reply:
fixed the issue, so I'm resolving the comment.